tests/unit/amo/components/TestSearchFilters.js

```diff
@@ -244,4 +244,10 @@ describe(__filename, () => {
       }),
     });
   });
+
+  it('does not display the addonType filter when a category is defined', () => {
```

Comment:
Referencing the issue here would be good, as this is unexpected behaviour (to me) and I think warrants explanation as to why we do it. Just a link should be good 😄